### ISSUE TITLE
[bug]: Focus ring too close to active audience link

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#6934]
### 🐞 Describe the bug

<img width="545" height="160" alt="Image" src="https://github.com/user-attachments/assets/49aa01ca-47e9-406a-b37d-bc51b481f13d" />

The focus ring on a non-active audience link should not touch the border of an active link. There should at least be a 2px space between.

**Comment by gfellerph** [comment]
Pattern discussion: consider focus rings when defining gaps. Where else do we have to adjust?

**Comment by myrtasakellariou** [comment]
There is also this case of the Microsite mobile header where the focus ring gets cut off. 

<img width="582" height="376" alt="Image" src="https://github.com/user-attachments/assets/1b007652-6dc2-46e6-9879-18fc9edad450" />

**Comment by gfellerph** [comment]
create design ticket for header focus style revision, focus ring on icon button should be oval shaped and not a circle.

**Comment by alizedebray** [comment]
Could use the solution described here: https://github.com/swisspost/design-system/issues/6924#issuecomment-3728519946

**Comment by alizedebray** [comment]
> There is also this case of the Microsite mobile header where the focus ring gets cut off.

Will be solved by [#6999](https://github.com/swisspost/design-system/issues/6999)

**Comment by gfellerph** [comment]
Closing parent task, sub-tasks remain open.
